source/_integrations/squeezebox.markdown

```diff
@@ -31,11 +31,15 @@ The Squeezebox music player ecosystem, which can be controlled through this inte
 1. One or more [Squeezebox compatible hardware or software players](https://lyrion.org/players-and-controllers/).
 2. One or more [Lyrion Music Servers or Logitech Media Servers (LMS)](https://lyrion.org/getting-started) with the Squeezebox players connected to these servers.
 
-{% include integrations/config_flow.md %}
+## Supported devices
 
-{% note %}
-This platform uses the web interface of the Lyrion Music Server (LMS) to send commands. The default port of the web interface is 9000. It is the same port that you use to access the LMS through your web browser.
-{% endnote %}
+The integration supports any [Squeezebox compatible hardware or software players](https://lyrion.org/players-and-controllers/) and both Lyrion Music Servers and Logitech Media Servers.
+
+## Data updates
```

Comment:
is this information needed in order to set up the integration? If not, I would place it further down in the doc.